evergreen.yml

```diff
@@ -347,11 +375,22 @@ functions:
     - command: s3.put
       params:
         build_variants:
-          - windows-64
+          - macos
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        local_file: mongosql-odbc-driver/target/release/atsql.pdb
-        remote_file: mongosql-odbc-driver/artifacts/${version_id}/${build_variant}/release/atsql.pdb
+        local_file: mongosql-odbc-driver/target/release/libatsql.dylib
+        remote_file: mongosql-odbc-driver/artifacts/${version_id}/${build_variant}/release/libatsql.dylib
```

Comment:
Do we have pdb files to upload as well?

Reply:
pdb files are windows

Reply:
Cool! 
Who doesn't like elf and dwarf, but elves and dwarfs themselves lol